modules/UI/UI.js

```diff
@@ -503,7 +503,10 @@ UI.updateUserRole = user => {
  * @param {string} status - The new status.
  */
 UI.updateUserStatus = (user, status) => {
-    if (!status) {
+    const reduxState = APP.store.getState() || {};
+    const { calleeInfoVisible } = reduxState['features/invite'] || {};
```

Comment:
Is it safe to prevent all status updates notifications when calleeInfoVisible is true? There isn't a case where you'd want the notification?

Reply:
As far as I know it should be safe because the notifications should be related to the callee mainly and they should be related to the callee's status which are displayed in the callee info overlay. Do you have in mind anything specific that needs to be displayed?